app/models/comment.rb

```diff
@@ -439,20 +439,58 @@ def self.check_and_add_tweets(tweets)
   end
 
   def self.get_node_from_urls_present(urls)
+    puts ">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>"
+    puts "all urls ->"
+    puts urls
+    puts ">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>"
     urls.each do |url|
+      puts ">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>"
+      puts "url.include? https:// -> #{url.include? "https://"}"
+      puts ">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>"
       next unless url.include? "https://"
 
       if url.last == "."
         url = url[0...url.length - 1]
       end
+
```

Comment:
And here (lines 452 and 453) you could use the **end_with?** and **chomp** methods

Reply:
Thanks for the review, @seabl!